v1/feed.js

```diff
@@ -193,7 +197,7 @@ Feed.prototype.aggregated = function(hyper, req) {
             }
 
             if (feed.random) {
-                feed.random.items = assignAllSummaries(feed.random.items);
+                feed.random = assignAllSummaries(feed.random.items);
```

Comment:
Uh, the Android app expects to have `feed.random.items`. Why this change in the interface?